pytket/docs/extensions.md

```diff
@@ -0,0 +1,131 @@
+
+[AQTBackend](https://cqcl.github.io/pytket-aqt/api/api.html#pytket.extensions.aqt.AQTBackend)
+\- Interface to an AQT device or simulator.
+
+[IQMBackend](inv:#*.IQMBackend)
+\- Interface to an IQM device or simulator.
+
+## Cloud Access
+
+[BraketBackend](inv:#*.BraketBackend)
+\- Interface to Amazon Braket service.
+
+## Emulators
+
+[IBMQEmulatorBackend](inv:#*.IBMQEmulatorBackend) - A backend which uses the [AerBackend](inv:#*.AerBackend) to emulate the behavior of IBMQBackend.
+
+[QuantinuumBackend](inv:#*.QuantinuumBackend)
+\- The QuantinuumBackend has two available emulators namely H1-1E and H2-1E. These are device specific emulators for the H1-1 and H2-1 devices. These emulators run remotely on a server.
+
+## Statevector Simulators
+
+[CirqStateSampleBackend](inv:#*.CirqStateSampleBackend)
+\- Backend for Cirq statevector simulator sampling.
+
+[CirqStateSimBackend](inv:#*.CirqStateSimBackend)
+\- Backend for Cirq statevector simulator state return.
+
+[AerStateBackend](inv:#*.AerStateBackend) - Backend for running simulations on the Qiskit Aer Statevector simulator.
+
+[ForestStateBackend](inv:#*.ForestStateBackend) - State-based interface to a Rigetti device.
+
+[ProjectQBackend](inv:#*.ProjectQBackend)
+\- Backend for running statevector simulations on the ProjectQ simulator.
+
+## Unitary Simulators
+
+[AerUnitaryBackend](inv:#*.AerUnitaryBackend) - Backend for running simulations on the Qiskit Aer unitary simulator.
+
+## Density Matrix Simulators
+
+[CirqDensityMatrixSampleBackend](inv:#*.CirqDensityMatrixSampleBackend)
+\- Backend for Cirq density matrix simulator sampling.
+
+[CirqDensityMatrixSimBackend](inv:#*.CirqDensityMatrixSimBackend)
+\- Backend for Cirq density matrix simulator density_matrix return.
+
+[QulacsBackend](inv:#*.QulacsBackend) - This has a configurable density matrix simulation option.
+
+Use `QulacsBackend(result_type="density_matrix")`.
+
+## Clifford Simulators
+
+[CirqCliffordSampleBackend](inv:#*.CirqCliffordSampleBackend)
+\- Backend for Cirq Clifford simulator sampling.
+
+[CirqCliffordSimBackend](inv:#*.CirqCliffordSimBackend)
+\- Backend for Cirq Clifford simulator state return.
+
+[SimplexBackend](inv:#*.SimplexBackend)- Backend for simulating Clifford circuits using pysimplex.
+
+[StimBackend](inv:#*.StimBackend)
+\- Backend for simulating Clifford circuits using Stim.
+
+## Other
+
+[AerBackend](inv:#*.AerBackend)
+\- Backend for running simulations on the Qiskit Aer QASM simulator. This simulator is noiseless by default but can take a user defined `NoiseModel`.
+
+[QulacsBackend](inv:#*.QulacsBackend)
+\- Backend for running simulations of variational quantum circuits on the Qulacs simulator.
+
+```{toctree}
+:caption: 'Extensions:'
+:maxdepth: 0
+
+pytket-aqt <https://cqcl.github.io/pytket-aqt/api/index.html>
+pytket-azure <https://docs.quantinuum.com/tket/extensions/pytket-azure>
+pytket-braket <https://docs.quantinuum.com/tket/extensions/pytket-braket>
```

Comment:
Unfortunately the linking syntax doesn't straightforwardly work with these TOCtree links which is annoying. I'll have to come back to this.